packages/scoutgame/src/protocol/calculateWeeklyClaims.ts

```diff
@@ -114,68 +111,128 @@ export async function calculateWeeklyClaims({
   const builderNfts = await prisma.builderNft.findMany({
     where: {
       chainId: scoutProtocolChainId,
-      contractAddress: nftContractAddress
+      season
     },
     select: {
+      contractAddress: true,
       tokenId: true,
       builderId: true,
       builder: {
         select: {
-          wallets: true
+          wallets: {
+            where: {
+              primary: true
+            },
+            select: {
+              address: true
+            }
+          }
         }
       }
     }
   });
 
+  const starterContractAddress = getStarterNFTContractAddress(week);
+  const standardContractAddress = getNFTContractAddress(week);
+
   const allClaims = await Promise.all(
-    topWeeklyBuilders
+    topWeeklyDevelopers
       // We only want to issue claims for builders that have sold at least one NFT
-      .filter(
-        (builder) =>
-          !!tokenBalances[builderNfts.find((nft) => nft.builderId === builder.builder.id)!.tokenId.toString()]
-      )
+      .filter((developer) => {
+        const starterNft = builderNfts.find(
```

Comment:
Two questions:
- Don't we need to still check that there is a token balance for their NFTs? Otherwise this is just going to include all developers.
- Can you just use nftType to determine starter vs default? This is better as it doesn't rely looking up the season configuration.